src/SearchResults.js

```diff
@@ -0,0 +1,48 @@
+import SearchResult from "./SearchResult";
+import moment from "moment";
+
+const SearchResults = props => {
+  return (
+    <div className="table-responsive">
+      <table class="table table-hover table-bordered">
+        <thead class="thead-dark">
+          <tr className="text-center">
+            <th scope="col">ID</th>
+            <th scope="col">Title</th>
+            <th scope="col">First Name</th>
+            <th scope="col">Last Name</th>
+            <th scope="col">Email</th>
+            <th scope="col">Room ID</th>
+            <th scope="col">Check in Date</th>
+            <th scope="col">Check out Date</th>
+            <th scope="col">Total Nights</th>
+            <th scope="col">Profiles</th>
+          </tr>
+        </thead>
+        <tbody>
+          {props.results.map(user => {
+            let a = moment(user.checkInDate, "YYYY-MM-DD");
+            let b = moment(user.checkOutDate, "YYYY-MM-DD");
```

Comment:
Think we can use `const` for these? Generally a better idea to use `const` if we don't plan on reassigning something.